### DISCUSSION TITLE
Collapsible text sections within a notebook page

### DISCUSSION THREAD

**Discussion by mbluett88** [zim-desktop-wiki/zim-desktop-wiki#2452] [Ideas]
Sometimes there is the need to have a lot of explanatory text which you don't want visible all the time.

Having collapsible sections within a Notebook page would be of great benefit.
+ icon  (expands)
- icon (collapses)

**Comment by introt** [comment]
Thanks for your input, afraid it's a duplicate of ~#1667~ #474 however
